Magento2/Tests/Eslint/AndSelfTest.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento2\Tests\Eslint;
+
+/**
+ * Class AndSelfTest
+ *
+ * Test Eslint Rule: jquery-no-andSelf.js
+ */
+class AndSelfTest extends AbstractEslintTestCase
+{
+    public function testExecute()
```

Comment:
And remaining functions as well
```suggestion
    public function testExecute(): void
```